plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/ERDPreferencesInitializer.java

```diff
@@ -35,7 +37,10 @@ public void initializeDefaultPreferences() {
         DBPPreferenceStore store = new BundlePreferenceStore(ERDUIActivator.getDefault().getBundle());
         PrefUtils.setDefaultPreferenceValue(store, ERDUIConstants.PREF_DIAGRAM_SHOW_VIEWS, true);
         PrefUtils.setDefaultPreferenceValue(store, ERDUIConstants.PREF_DIAGRAM_SHOW_PARTITIONS, false);
-        PrefUtils.setDefaultPreferenceValue(store, ERDUIConstants.PREF_ROUTING_TYPE, ERDUIConstants.ROUTING_SHORTEST_PATH);
+        ERDConnectionRouterDescriptor defaultRouter = ERDConnectionRouterRegistry.getInstance().getDefaultRouter();
```

Comment:
It is better to keep prefs initializer as simple as possible (because it is invoked on a very early stage of plugin initialization). just set it it to constant value.
This constant can be reset in another plugin.

Reply:
Done